frontend/nuxt.config.ts

```diff
@@ -68,14 +68,14 @@ export default defineNuxtConfig({
   },
   /**
    * Robots.txt rules are configured here via the \@nuxtjs/robots package.
-   * @see {@link https://nuxtseo.com/robots/guides/nuxt-config|Robots Config Rules}
+   * @see {@link https://nuxtseo.com/robots/guides/nuxt-config}
    */
   robots: {
-    disallow: ["/search", "/search/audio", "/search/image"],
+    disallow: ["/search"],
```

Comment:
Robots rules are prefix based. As such, there's no need to define `/search/audio` (and image) because `/search` already matches everything that starts with `/search`.